### PR TITLE
Agregando las 'clausulas para invitar' a @leonel y resolviendo algunos Issues

### PR DESCRIPTION
**Pull request by josuedanielbust**

### CODE REVIEW

**Comment by josuedanielbust** [comment]
Second pull request try to solve issue #36 with the comment of Yeiner

**Comment by josuedanielbust** [comment]
Third push corresponding to #39 issue

File: index.js [hunk 1]
```diff
@@ -75,12 +75,52 @@ controller.hears(['help', 'ayuda'], ['direct_message', 'direct_mention'], (bot,
...
+})
+
+/**
+ * BairesDev
+ */
+controller.hears(['baires', 'bairesdev', 'BairesDev', 'bairesDev'], ['direct_message', 'direct_mention'], (bot, message) => {
+  let bairesdev = [
+    '¿Me preguntas por BairesDev?, esto es lo que yo se:',
+    '- Tienen un bot de Spamming (Que nunca arreglan) :troll:',
+    '- Son falta de respuesta.',
```
Comment (buritica) [inline index.js]: Creo que esto esta mal redactado :P

File: index.js [hunk 2]
```diff
@@ -75,12 +75,52 @@ controller.hears(['help', 'ayuda'], ['direct_message', 'direct_mention'], (bot,
...
+    '- mira `@leonel clausulas` si no podés invitar a tu parcero.',
     `... y me podés estender en ${packageInfo.homepage}`
   ].join('\n')
   bot.reply(message, help)
 })
 
 /**
+ * Clausulas
+ */
+controller.hears(['clausulas', 'clausulas', 'clause', 'why'], ['direct_message', 'direct_mention'], (bot, message) => {
```
Comment (buritica) [inline index.js]: pruebas :D

**Review by buritica — CHANGES_REQUESTED** [review]
Se ve bien, si le agregas pruebas le hago merge :P

**Comment by josuedanielbust** [comment]
@buritica I think that I need some help for make the tests, I search on docs for learn to do but I don't know where start or what I need really do 😣

**Comment by buritica** [comment]
did you check the tests that are already there?

give it a try and I'll help you out

**Comment by josuedanielbust** [comment]
@buritica what do you think about this test?

**Comment by buritica** [comment]
hey @JosueDanielBust, thank you for sending this PR, unfortunately I'm going to close it for the following reasons:

- the "clausulas" are a good idea, but they are the wrong solution for a UX problem. What I've done instead is improve the messages people get when something goes wrong during the invitation process #83

- the baires command seemed like a good idea at the time. After some thought, I think it's best if col-dev stays away from an official message about any company, even if they have really bad practices.

- the `coqueto` command is meant to be an easter egg, so we'd rather not have it in the help menu.

Thanks again, I can tell you did some really good work and invested your time. I can also see it helped you learn how to test, which is not easy to do! I'm sorry it took so long to get to this PR, if you have any new ideas, let me know and maybe we can discuss them together before you put any time into them, I'd hate for you to feel like you've wasted your time!

**Pull request closed without merging**